app/dao/inbound_sms_dao.py

```diff
@@ -43,31 +40,29 @@ def dao_get_paginated_inbound_sms_for_service_for_public_api(
 
     filters = [InboundSms.service_id == service_id]
 
-    if older_than:
-        older_than_created_at = db.session.query(
-            InboundSms.created_at).filter(InboundSms.id == older_than).as_scalar()
-        filters.append(InboundSms.created_at < older_than_created_at)
+    if older_than is not None:
```

Comment:
I think "is not None" makes sense here.  "older_than" should be a datetime instance or None.

Reply:
reverted